gradle/groovyProject.gradle

```diff
@@ -49,12 +65,15 @@ testTasks.all { task ->
             }
         }
     }
-    if (javaVersion.java7) {
+    executable = file("${javaInstallationForTest.javaHome}/bin/java")
```

Comment:
❌ I guess this will not work on Windows. You can use `Jvm.forHome(file(java7Home)).javaExecutable`.

Reply:
Thanks, fixed